redisinsight/electron/preload.ts

```diff
@@ -0,0 +1,26 @@
+// Disable no-unused-vars, broken for spread args
+/* eslint no-unused-vars: off */
+import { contextBridge, ipcRenderer } from 'electron';
+import { IpcEvent } from 'uiSrc/electron/constants';
+
+const electronHandler = {
+  ipcRenderer: {
+    invoke: (channel: IpcEvent, data?: any) => {
+      // whitelist channels
+      if (Object.values(IpcEvent).includes(channel)) {
+        return ipcRenderer.invoke(channel, data);
+      }
+
+      return new Error('channel is not allowed');
+    },
+  },
+};
+
+contextBridge.exposeInMainWorld('electron', electronHandler);
+
+contextBridge.exposeInMainWorld('ENV_VARS', {
```

Comment:
I don't like to export env variables. Let's have electron.config object with everything needed. What do you think?